DisCatSharp/Enums/Invite/TargetActivity.cs

```diff
@@ -45,19 +45,19 @@ public enum TargetActivity : ulong
 	ChessInThePark = 832012774040141894,
 
 	/// <summary>
-	/// Represents the embedded application Doodle Crew.
+	/// Represents the embedded application Fishington.io.
```

Comment:
this file should be removed a article written with the activities / or only prod apps kept